cli/running/instance.go

```diff
@@ -50,6 +50,12 @@ type Instance struct {
 	waitMutex sync.Mutex
 	// done represent whether the instance was stopped.
 	done bool
+	// stateMutex used to avoid a race condition under started and shouldStop fields.
+	stateMutex sync.Mutex
+	// started indicates whether the Watchdog has started an Instance.
+	started bool
```

Comment:
If this is a watchdog flag, it must part of watchdog struct. If not, this state flag must be controlled by instance functions.
Can I use Instance struct without watchdog? If yes, there is no point to mention it in the comment of these flags: Instance is not dependent on watchdog.